Resources/Prototypes/DeltaV/Entities/Objects/Decoration/flora.yml

```diff
@@ -0,0 +1,100 @@
+- type: entity
+  id: BushBase
+  name: Bush
```

Comment:
names should be lowercase since they arent proper nouns

Reply:
👍